src/backend/columnar/Makefile

```diff
@@ -0,0 +1,22 @@
+safestringlib_srcdir = $(citus_abs_top_srcdir)/vendor/safestringlib
```

Comment:
Also need a line:

`citus_subdir = src/backend/columnar`

At the top of this Makefile.